src/osgEarth/ObjectIndex.cpp

```diff
@@ -63,7 +63,11 @@ ObjectIndex::ObjectIndex() :
 _idGen( STARTING_OBJECT_ID )
 {
     _attribName     = "oe_index_objectid_attr";
+#if defined(OSG_GLES3_AVAILABLE)
+    _attribLocation = osg::Drawable::TEXTURE_COORDS_5; // Why is SECONDARY_COLORS occupied. Could it actually be a QT issue?
```

Comment:
This is also an unresolved issue on OSX. Does it also happen on IOS and Android?

Reply:
Glenn, this is solved? Here: https://github.com/gwaldron/osgearth/issues/1098

Reply:
Oh right ... forgot about that. Thanks Remo :)

Reply:
Good to hear i was going to ask as it seemed this  might be adustable from the application level also